src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -70,31 +75,138 @@ protected ActionResult execute() throws EntityDoesNotExistException{
+        }
+        generateStatusMessage(data, logs);
+        data.init(ifShowAll, ifShowTestData, logs);
         
-        if (offset == null) {
+        if (searchTimeOffset.isEmpty()) {
             return createShowPageResult(Const.ViewURIs.ADMIN_ACTIVITY_LOG, data);
         }
         
         return createAjaxResult(data);
     }
     
-    private LogQuery buildQuery(String offset, AdminActivityLogPageData data) {
+    private void generateStatusMessage(AdminActivityLogPageData data, List<ActivityLogEntry> logs) {
+        String status = "Total Logs gone through in last search: " + totalLogsSearched + "<br>";
+        status += "Total Relevant Logs found in last search: " + logs.size() + "<br>";
+        
+        long earliestSearchTime = data.getFromDate();
+        ActivityLogEntry earliestLogChecked = null;
+        if (!logs.isEmpty()) {
+            earliestLogChecked = logs.get(logs.size() - 1);
+        }
+        //  if the search space is limited to a certain log
+        if ((logs.size() >= RELEVANT_LOGS_PER_PAGE) && (earliestLogChecked != null)) {
+            earliestSearchTime = earliestLogChecked.getTime();
+        }
+        
+        double targetTimeZone = Const.DOUBLE_UNINITIALIZED;
+        if (data.isPersonSpecified()) {
+            String targetUserGoogleId = data.getPersonSpecified();
+            targetTimeZone = getLocalTimeZoneForRequest(targetUserGoogleId, "");
+        }
+        
+        if (targetTimeZone == Const.DOUBLE_UNINITIALIZED) { // if no person is specified or the person doesn't really exist
+            if ((logs.size() >= RELEVANT_LOGS_PER_PAGE) && (earliestLogChecked != null)) {
+                String userGoogleId = earliestLogChecked.getGoogleId();
+                String userRole = earliestLogChecked.getRole();
+                targetTimeZone = this.getLocalTimeZoneInfo(userGoogleId, userRole);
+            } else {
+                targetTimeZone = Const.SystemParams.ADMIN_TIMZE_ZONE_DOUBLE;
+            }
+        }
+        
+        double adminTimeZone = Const.SystemParams.ADMIN_TIMZE_ZONE_DOUBLE;
+        String timeInAdminTimeZone = computeLocalTime(adminTimeZone, String.valueOf(earliestSearchTime));
+        String timeInUserTimeZone =  computeLocalTime(targetTimeZone, String.valueOf(earliestSearchTime));
+        status += "The earliest log entry checked on <b>" + timeInAdminTimeZone + "</b> in Admin Time Zone (" 
+                  + adminTimeZone + ") and ";
+        if (targetTimeZone != Const.DOUBLE_UNINITIALIZED) {
+            status += "on <b>" + timeInUserTimeZone + "</b> in Local Time Zone (" + targetTimeZone + ").<br>";
+        } else {
+            status += timeInUserTimeZone;
+        }
+        
+        // the "Search More" button to continue searching from the previous fromDate 
+        status += "<button class=\"btn-link\" id=\"button_older\" onclick=\"submitFormAjax(" + data.getFromDate() + ");\">Search More</button>";
```

Comment:
do you think there will be duplicates if we set the endtime of the second search to be the starttime of the first?